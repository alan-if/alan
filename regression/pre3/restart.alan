Syntax q = 'q'.
Verb q
  Does
    Quit.
End Verb.

Syntax restart = 'restart'.
Verb restart
  Does
    Restart.
End Verb.

Syntax 'score' = 'score'.
Verb 'score'
  Does
    Score.
End Verb.

Location l1
  Exit w To l2
    Does
      Score 5.
  End Exit.
End Location.

Location l2
End Location.

Start At l1.
  "Going w will give you 5 points. Then restart (or q) to find out if
	scores are cleared."

