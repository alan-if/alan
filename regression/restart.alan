Syntax q = 'q'.
Verb q
  Does
    Quit.
End Verb.

Syntax 'score' = 'score'.
Verb 'score'
  Does
    Score.
End Verb.

Location l1
  Exit w To l2
    Does
      Score 5.
  End Exit.
End Location.

Location l2
End Location.

Start At l1.
  "Going w will give you 5 points. Then restart (q) to find out if
	scores are cleared."

