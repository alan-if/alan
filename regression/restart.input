w
score
q
y
score
