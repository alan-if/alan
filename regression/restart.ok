
Going w will give you 5 points. Then restart (q) to find out if scores
are cleared.


L1.

> L2.

> You have scored 5 points out of 5.

> 
Do you want to restart (y) ? 
Going w will give you 5 points. Then restart (q) to find out if scores
are cleared.


L1.

> You have scored 0 points out of 5.

> 

Do you want to restart (y) ? 
