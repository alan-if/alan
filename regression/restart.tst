../compiler/alan restart
../interpreter/arun -n restart < restart.input
