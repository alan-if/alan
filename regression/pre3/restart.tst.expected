########## restart ##########
../compiler/alan restart
../interpreter/arun -n restart < restart.input
########## /bin/sh  restart.tst ##########

        No warnings or errors detected.
        3 informational message(s).


Going w will give you 5 points. Then restart (or q) to find out if scores
are cleared.


L1

> w
L2

> score
You have scored 5 points out of 5.

> restart

Are you sure (RETURN confirms) ? 

Going w will give you 5 points. Then restart (or q) to find out if scores
are cleared.


L1

> score
You have scored 0 points out of 5.

> 

Do you want to RESTART, RESTORE or QUIT ? 
